### PR TITLE
[travis] fix for jshint version

### PR DESCRIPTION
**Pull request by damianz5**

### CODE REVIEW

**Comment by damianz5** [comment]
It's better now but still some issues in two tests
https://travis-ci.org/ezsystems/PlatformUIBundle/jobs/297389290

File: bin/travis/setupnode.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+npm install -g npm
```
Comment (sunpietro) [inline bin/travis/setupnode.sh]: Just curious. You'are asking npm to install itself globally, right?

File: bin/travis/setupnode.sh [hunk 2]
```diff
@@ -13,6 +15,8 @@ npm install -g phantomjs@1.9.8
 echo "> installing global packages"
 npm install -g grunt-cli grover bower
 
+npm install -g jshint@2.9.5
```
Comment (sunpietro) [inline bin/travis/setupnode.sh]: I think it should install the latest jshint, not the specific version.

File: bin/travis/setupnode.sh (hunk 1 above)
Comment (damianz5) [reply bin/travis/setupnode.sh]: There were some version differences, but after tests I can remove this line

**Comment by dew326** [comment]
To avoid the issue in tests try to remove esversion:6 from the file and add to the jshint config:
https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/services/ez-locationviewviewservice.js#L5
https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/services/ez-navigationhubviewservice.js#L5

and add the new rule in config: `esverion: 6`
https://github.com/ezsystems/PlatformUIBundle/blob/master/jshint.json

**Comment by damianz5** [comment]
@sunpietro can you check the two failed tests? [L779](https://travis-ci.org/ezsystems/PlatformUIBundle/jobs/297850932#L779) and [L785](https://travis-ci.org/ezsystems/PlatformUIBundle/jobs/297850932#L785)

**Comment by damianz5** [comment]
@dew326 it's out of scope of this PR - the test environment is ready to use, please fix the tested code in separate PR

**Comment by andrerom** [comment]
If we merge this, can someone take care of adapting the failing tests? From what I can see they haven't changes since 2016, so the break is caused by updated does or nod, or?

**Comment by sunpietro** [comment]
@andrerom I'll correct them after merging this PR

**Comment by andrerom** [comment]
Ok, @damianz5 could you change this to either 1.11 or 1.12 so we can merge?

**Comment by damianz5** [comment]
@andrerom done

**Comment by andrerom** [comment]
Thanks @damianz5, merged, all yours @sunpietro :D ;)

**Pull request merged**